teapots/common/ndk_helper/CMakeLists.txt

```diff
@@ -1,16 +1,35 @@
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11 -Wall -fno-exceptions -fno-rtti")
 
-add_library(ndk-helper STATIC
-            gestureDetector.cpp
-            gl3stub.cpp
-            GLContext.cpp
-            interpolator.cpp
-            JNIHelper.cpp
-            perfMonitor.cpp
-            sensorManager.cpp
-            shader.cpp
-            tapCamera.cpp
-            vecmath.cpp)
+add_library(ndkHelper
+  STATIC
+    gestureDetector.cpp
+    gl3stub.cpp
+    GLContext.cpp
+    interpolator.cpp
+    JNIHelper.cpp
+    perfMonitor.cpp
+    sensorManager.cpp
+    shader.cpp
+    tapCamera.cpp
+    vecmath.cpp
+)
 
-target_include_directories(ndk-helper PRIVATE
-                           ${ANDROID_NDK}/sources/android/native_app_glue)
+set_target_properties(ndkHelper
+  PROPERTIES
+    CXX_STANDARD 11
+    CXX_STANDARD_REQUIRED YES
+    CXX_EXTENSIONS NO
+)
+target_compile_features(ndkHelper
+  PUBLIC
+    cxx_noexcept
+    cxx_nullptr
```

Comment:
What does this do? Is it really necessary. I would expect nullptr to be subsumed by CXX_STANDARD 11